dashboard/containers/dispatcher/fontprojects.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+import csv
+import urllib
+CACHE_CSV = "fontprojects.csv"
```

Comment:
I do store the data on disk on purpose. So that I can keep track of history on the git repo. I'm OK with fetching data from a remote spreadsheet, but I also like the guarantee of having access to the data even if the spreadsheet goes offline.